cmd/ci-operator/main.go

```diff
@@ -503,6 +508,7 @@ func (o *options) Complete() error {
 		o.jobSpec.Refs.PathAlias = *config.CanonicalGoRepository
 	}
 	o.configSpec = config
+	o.jobSpec.Metadata = config.Metadata
```

Comment:
So there's a class of jobs (looking at you, release informers...) which have neither Prow refs nor metadata as they use `$RESOLVED_CONFIG` or whatever and don't actually come from the server. We can phase in requiring those jobs to explicitly set `zz_generated_metadata` manually so that their Pods end up getting automatic requests & limits?

Reply:
Shouldn't be too difficult to make a script that adds `zz_generated_metadata` to all `CONFIG_SPEC` and `UNRESOLVED_CONFIG` sections for handwritten release jobs. I don't think we need anything more complicated for those jobs.